flexget/plugins/input/my_anime_list.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+
+import logging
+import xml.etree.ElementTree as ET
+
+from flexget import plugin
+from flexget.config_schema import one_or_more
+from flexget.entry import Entry
+from flexget.event import event
+from flexget.utils.cached_input import cached
+from flexget.utils.requests import RequestException
+
+log = logging.getLogger('my_anime_list')
+STATUS = {
+    '1': 'watching',
+    '2': 'completed',
+    '3': 'on_hold',
+    '4': 'dropped',
+    '6': 'plan_to_watch',
+}
+
+ANIME_TYPE = {
+    '1': 'series',
+    '2': 'ova',
+    '3': 'movie',
+    '4': 'special',
+    '5': 'ona'
+}
+
+
+class MyAnimeList(object):
+    """" Creates entries for series and movies from MyAnimeList list
+    Syntax:
+    my_anime_list:
+      username: <value>
+      status:
+      	- <watching|completed|on_hold|dropped|plan_to_watch>
+      	- <watching|completed|on_hold|dropped|plan_to_watch>
+        ...
+    """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'username': {'type': 'string'},
+            'status': one_or_more({'type': 'string', 'enum': ['watching', 'completed', 'on_hold', 'dropped', 'plan_to_watch']}),
+            'type': one_or_more({'type': 'string', 'enum': ['series', 'ova', 'movie', 'ona']})
+        },
+        'required': ['username'],
+        'additionalProperties': False
+    }
+
+    @cached('my_anime_list', persist='2 hours')
+    def on_task_input(self, task, config):
+        entries = []
+        parameters = {'u': config['username'], 'status': 'all', 'type': 'anime'}
```

Comment:
Oh yeah, this part of their API is a bit stupid, since it does not do anything with the status parameter. I can technically exclude it, it just got left them there to be safe.